installers/rpm-code/codes/srw.sh

```diff
@@ -33,43 +33,51 @@ srw_main() {
 
 srw_python_install() {
     install_pip_install primme
-    cd SRW/cpp/py
```

Comment:
srw_main does not need to do the make. That's done by srw_python_install.

Also, please verify that the pip installed *.so correctly links to sfftw.

Reply:
> srw_main does not need to do the make. That's done by srw_python_install.

Please explain this more, do you mean I should remove line 31?

> Also, please verify that the pip installed *.so correctly links to sfftw.

How can I verify this?

Reply:
> Please explain this more, do you mean I should remove line 31?

Lines 25-31. pip install is doing the make in its setup.py.

> How can I verify this?

This is with the current rpm:

```sh
[py3;@v site-packages]$ ldd srwlpy.so
ldd: warning: you do not have execution permission for `./srwlpy.so'
        linux-vdso.so.1 (0x00007ffd4a76c000)
        libfftw3f.so.3 => /lib64/libfftw3f.so.3 (0x00007f2ed9600000)
        libfftw3.so.3 => /lib64/libfftw3.so.3 (0x00007f2ed940c000)
        libstdc++.so.6 => /lib64/libstdc++.so.6 (0x00007f2ed9000000)
        libm.so.6 => /lib64/libm.so.6 (0x00007f2ed932e000)
        libgcc_s.so.1 => /lib64/libgcc_s.so.1 (0x00007f2ed930e000)
        libc.so.6 => /lib64/libc.so.6 (0x00007f2ed8c00000)
        /lib64/ld-linux-x86-64.so.2 (0x00007f2ed99fc000)
```

libfftw3f is the single precision library. I'm not sure what's doing the conversion from sfftw to fftw3f. I think we need to look to see if the makefile is getting generated correctly with the old version; does sfttw actually get injected with the perl command or is there a bug here that means setup.py does the right thing?

Reply:
Let's meet to discuss this and the patches.

Reply:
https://github.com/radiasoft/download/commit/994d56ec4eb7008c310ccbcf15d10cae03f86f38